### PR TITLE
ci: disable rook deployed csi drivers to speed up e2e

### PR DESCRIPTION
**Pull request by Rakshith-R**
Signed-off-by: Rakshith R <rar@redhat.com>

---

<details>
<summary>Show available bot commands</summary>

These commands are normally not required, but in case of issues, leave any of
the following bot commands in an otherwise empty comment in this PR:

- `/retest ci/centos/<job-name>`: retest the `<job-name>` after unrelated
  failure (please report the failure too!)
- `/retest all`: run this in case the CentOS CI failed to start/report any test
  progress or results

</details>

### CODE REVIEW

**Review by humblec — DISMISSED** [review]

**Review by yati1998 — DISMISSED** [review]

File: scripts/rook.sh [hunk 1]
```diff
@@ -42,22 +42,28 @@ function deploy_rook() {
 	  then
 	      kubectl_retry create -f "${ROOK_URL}/crds.yaml"
 	  fi
+        TEMP_DIR="$(mktemp -d)"
+        curl -o "${TEMP_DIR}/operator.yaml" "${ROOK_URL}/operator.yaml"
+        # disable rook deployed csi drivers
+        sed -i "s|ROOK_CSI_ENABLE_CEPHFS: \"true\"|ROOK_CSI_ENABLE_CEPHFS: \"false\"|g" "${TEMP_DIR}/operator.yaml"
```
Comment (nixpanic) [inline scripts/rook.sh]: please drop the escaping of quotes to make it more readable, use `'` in the outer string:
```
sed -i 's|ROOK_CSI_ENABLE_CEPHFS: "true"|ROOK_CSI_ENABLE_CEPHFS: "false"|g' "${TEMP_DIR}/operator.yaml"
````

**Review by nixpanic — CHANGES_REQUESTED** [review]

**Review by nixpanic — APPROVED** [review]

**Review by Madhu-1 — APPROVED** [review]

**Review by yati1998 — APPROVED** [review]

**Comment by Rakshith-R** [comment]
> /retest ci/centos/mini-e2e-helm/k8s-1.20

failed due to timeout.
https://jenkins-ceph-csi.apps.ocp.ci.centos.org/blue/organizations/jenkins/mini-e2e-helm_k8s-1.20/detail/mini-e2e-helm_k8s-1.20/2796/pipeline

**Comment by nixpanic** [comment]
added `[DNM]` label so that #2571 can get merged as early as possible

**Comment by nixpanic** [comment]
@mergifyio refresh

**Pull request merged**